app/assets/javascripts/discourse/app/components/bootstrap-mode-notice.js

```diff
@@ -1,11 +1,25 @@
+import { action } from "@ember/object";
 import { inject as service } from "@ember/service";
 import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import DiscourseURL from "discourse/lib/url";
+import User from "discourse/models/user";
 
 export default class BootstrapModeNotice extends Component {
   @service siteSettings;
 
-  get href() {
-    const topicId = this.siteSettings.admin_quick_start_topic_id;
-    return `/t/-/${topicId}`;
+  @tracked showUserTip = false;
+
+  @action
+  setupUserTip() {
+    this.showUserTip = User.current().canSeeUserTip("admin_guide");
```

Comment:
I think we prefer to use:

@service currentUser;

and in your function here: `this.currentUser`